addon/bin/sign

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+const version = require('../package.json').version;
+const resolve = require('path').resolve;
+const xpiPath = resolve(`dist/testpilot-addon-${version}.xpi`);
```

Comment:
If you want to use the version in the filename, I have a small `sed` command in our Makefile to extract the version number from install.rdf (assume specifically-formatted XML, which shouldn't be a big problem):

https://github.com/mozilla/universal-search/blob/master/Makefile#L3